toucan_connectors/mssql/mssql_connector.py

```diff
@@ -90,6 +89,8 @@ def get_form(cls, connector: "MSSQLConnector", current_config):
                         constraints["table"] = strlist_to_enum("table", available_tables, None)
 
                     cursor.close()
+        except Exception as exc:
+            getLogger(__name__).warning(f"unable to fetch database and/or table lists for {__name__}: {exc}")
```

Comment:
Please declare a top-level `_LOGGER` variable, and please use `exc_info` for exceptions, it will allow us to have a proper stack trace:

```suggestion
            _LOGGER.warning(f"unable to fetch database and/or table lists for {__name__}", exc_info=exc)
```

Reply:
[607ba51](https://github.com/ToucanToco/toucan-connectors/pull/1966/commits/607ba5129f8a8edd85ebe6761826559fe26786a8)